feature-libs/product-configurator/rulebased/components/attribute/header/configurator-attribute-header.component.spec.ts

```diff
@@ -119,7 +119,6 @@ describe('ConfigAttributeHeaderComponent', () => {
     productConfigurator: {
       enableNavigationToConflict: false,
       descriptions: {
-        addDescriptions: true,
```

Comment:
Could you please fix the strict mode issues in line 1126 and 1132? Last week I did notice them in my review